src/vs/workbench/parts/terminal/test/electron-browser/terminalLinkHandler.test.ts

```diff
@@ -41,6 +65,7 @@ suite('Workbench - TerminalLinkHandler', () => {
 			testLink('c:/a/long/path');
 			testLink('c:\\a\\long\\path');
 			testLink('c:\\mixed/slash\\path');
+			testLink('a/relative/path');
```

Comment:
Will this also linkify just plain words for example `"foo"`? That way after validation the output of `ls` should all be linked. If so a test for that would be great.